packages/openbridge-webcomponents/src/navigation-instruments/readout-advice/readout-advice.css

```
@@ -0,0 +1,32 @@
+
+.readout-advice-root {
+  width: 100%;
+  max-width: 100%;
+}
+
+.readout-advice-wrapper {
+  width: 100%;
+}
+
+.readout-advice-root.state-amplified {
+  @mixin style style=amplified visibleWrapperClass=.readout-advice-wrapper;
```

Stylelint (17.7.0)

[error] Expected mixin name "style style=amplified visibleWrapperClass=.readout-advice-wrapper" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)